backend/src/main/java/saci/domain/service/validators/LevelValidator.java

```diff
@@ -0,0 +1,19 @@
+package saci.domain.service.validators;
+
+import lombok.NoArgsConstructor;
+import saci.domain.model.Level;
+
+@NoArgsConstructor
+public final class LevelValidator {
+    public static boolean ValidMaxCoefficient(Level newLevel) {
```

Comment:
```suggestion
    public static boolean MaxCoefficientIsValid(Level newLevel) {
```

Try to define your method names as clear as possible